icons/fedora/fedora-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128"><path style="stroke:none;fill-rule:nonzero;fill:#294172;fill-opacity:1" d="M127.82 64.004C127.82 28.754 99.246.18 64 .18 28.766.18.203 28.73.18 63.957v49.39c.02 7.997 6.504 14.473 14.508 14.473h49.335c35.239-.015 63.797-28.578 63.797-63.816"/><path style="fill:none;stroke-width:29.21;stroke-linecap:butt;stroke-linejoin:miter;stroke:#3c6eb4;stroke-opacity:1;stroke-miterlimit:4" d="M77.123 142.096h47.846v47.847c0 26.408-21.43 47.846-47.846 47.846-26.409 0-47.838-21.438-47.838-47.846 0-26.409 21.43-47.847 47.838-47.847Zm0 0" transform="scale(.4794)"/><path style="fill:none;stroke-width:29.21;stroke-linecap:butt;stroke-linejoin:miter;stroke:#3c6eb4;stroke-opacity:1;stroke-miterlimit:4" d="M77.125 142.102h47.846v47.838c0 26.408-21.438 47.846-47.846 47.846-26.408 0-47.846-21.43-47.846-47.846 0-26.408 21.438-47.838 47.846-47.838Zm0 0" transform="rotate(180 59.856 68.144) scale(.4794)"/><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:1" d="M66.926 61.137v29.89c0 16.54-13.41 29.953-29.95 29.953-2.511 0-4.296-.285-6.617-.89-3.39-.887-6.156-3.664-6.16-6.895 0-3.906 2.836-6.746 7.074-6.746 2.016 0 2.747.387 5.704.387 8.718 0 15.793-7.063 15.808-15.785V77.312c0-1.23-1-2.23-2.234-2.226l-10.387-.004c-3.867 0-6.996-3.086-6.996-6.965 0-3.906 3.16-6.98 7.07-6.98"/><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:1" d="M52.785 75.148V45.262c0-16.543 13.41-29.953 29.953-29.953 2.508 0 4.293.28 6.617.89 3.387.887 6.157 3.664 6.157 6.895 0 3.906-2.836 6.746-7.07 6.746-2.02 0-2.75-.387-5.704-.387-8.722 0-15.797 7.063-15.812 15.781v13.743a2.235 2.235 0 0 0 2.234 2.226l10.387.004c3.871 0 6.996 3.086 6.996 6.965.004 3.906-3.16 6.98-7.07 6.98"/><path style="stroke:none;fill-rule:nonzero;fill:#3c6eb4;fill-opacity:1" d="M116.809 116.773v-2.652l-1.211 2.781-1.18-2.78v2.651h-.68v-4.187h.711l1.168 2.676 1.149-2.676h.722v4.187Zm-4.954-3.484v3.484h-.71v-3.484h-1.192v-.703h3.09v.703"/></svg>
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128"><path style="stroke:none;fill-rule:nonzero;fill:#294172;fill-opacity:1" d="M127.82 64.004C127.82 28.754 99.246.18 64 .18 28.766.18.203 28.73.18 63.957v49.39c.02 7.997 6.504 14.473 14.508 14.473h49.335c35.239-.015 63.797-28.578 63.797-63.816"/><path style="fill:none;stroke-width:29.21;stroke-linecap:butt;stroke-linejoin:miter;stroke:#3c6eb4;stroke-opacity:1;stroke-miterlimit:4" d="M77.123 142.096h47.846v47.847c0 26.408-21.43 47.846-47.846 47.846-26.409 0-47.838-21.438-47.838-47.846 0-26.409 21.43-47.847 47.838-47.847Zm0 0" transform="scale(.4794)"/><path style="fill:none;stroke-width:29.21;stroke-linecap:butt;stroke-linejoin:miter;stroke:#3c6eb4;stroke-opacity:1;stroke-miterlimit:4" d="M77.125 142.102h47.846v47.838c0 26.408-21.438 47.846-47.846 47.846-26.408 0-47.846-21.43-47.846-47.846 0-26.408 21.438-47.838 47.846-47.838Zm0 0" transform="rotate(180 59.856 68.144) scale(.4794)"/><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:1" d="M66.926 61.137v29.89c0 16.54-13.41 29.953-29.95 29.953-2.511 0-4.296-.285-6.617-.89-3.39-.887-6.156-3.664-6.16-6.895 0-3.906 2.836-6.746 7.074-6.746 2.016 0 2.747.387 5.704.387 8.718 0 15.793-7.063 15.808-15.785V77.312c0-1.23-1-2.23-2.234-2.226l-10.387-.004c-3.867 0-6.996-3.086-6.996-6.965 0-3.906 3.16-6.98 7.07-6.98"/><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:1" d="M52.785 75.148V45.262c0-16.543 13.41-29.953 29.953-29.953 2.508 0 4.293.28 6.617.89 3.387.887 6.157 3.664 6.157 6.895 0 3.906-2.836 6.746-7.07 6.746-2.02 0-2.75-.387-5.704-.387-8.722 0-15.797 7.063-15.812 15.781v13.743a2.235 2.235 0 0 0 2.234 2.226l10.387.004c3.871 0 6.996 3.086 6.996 6.965.004 3.906-3.16 6.98-7.07 6.98"/><path style="stroke:none;fill-rule:nonzero;fill:#3c6eb4;fill-opacity:1" d="M116.809 116.773v-2.652l-1.211 2.781-1.18-2.78v2.651h-.68v-4.187h.711l1.168 2.676 1.149-2.676h.722v4.187Zm-4.954-3.484v3.484h-.71v-3.484h-1.192v-.703h3.09v.703"/></svg>

```